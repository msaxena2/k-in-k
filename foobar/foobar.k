module FOOBAR
  syntax Foo ::= "bar" [klabel(bar), function]
               | "buzz" [klabel(buzz)]
               | "fizz" [klabel(fizz)]
  rule bar => buzz
  rule fizz => buzz
endmodule

