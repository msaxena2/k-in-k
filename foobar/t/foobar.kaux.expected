<T>
  <k>
    #done
  </k>
  <kore>
    <definitionAttributes>
      [ .Patterns ]
    </definitionAttributes>
    <modules>
      <koreModule>
        <name>
          FOOBAR
        </name>
        <sortDeclarations>
          SetItem ( sort Foo { .Names } [ .Patterns ] )
        </sortDeclarations>
        <symbolDeclarations>
          SetItem ( symbol bar { .Names } ( .Sorts ) : Foo [ .Patterns ] )
          SetItem ( symbol buzz { .Names } ( .Sorts ) : Foo [ .Patterns ] )
        </symbolDeclarations>
      </koreModule>
    </modules>
  </kore>
</T>
